### PR TITLE
Add analytics

### PR DESCRIPTION
**Pull request by dspencer001**

### CODE REVIEW

File: client/js/components/book/page.jsx [hunk 1]
```diff
@@ -62,13 +63,102 @@ export class Page extends React.Component {
     window.addEventListener("message", (e) => this.onMessage(e), false);
   }
 
+  /* We need to listen to video/audio play and pause events and click events on
+   * images inside the iframe.
+   */
+  addIframeEventListeners() {
+    let iframeDocument = this.contentIframe.contentDocument ||
+        this.contentIframe.contentWindow.document;
+
```
Comment (nick-benoit14) [inline client/js/components/book/page.jsx:72]: Does this only work if the iframe content is being served from the same domain?

File: client/js/components/book/page.jsx [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
-import React        from "react";
-import { connect }  from "react-redux";
+import _                          from "lodash";
+import React                      from "react";
+import { connect }                from "react-redux";
 
-import * as ContentActions  from "../../actions/content";
-import assets               from "../../libs/assets";
+import * as ContentActions        from "../../actions/content";
+import * as AnalyticsActions from "../../actions/analytics";
```
Comment (nick-benoit14) [inline client/js/components/book/page.jsx:8]: Maybe line up all the import statements

**Review by nick-benoit14 — COMMENTED** [review]
I can't remember if we decided that the content player already has to live on the same domain as the player. If not, do we know that attaching those listeners will work if the content player and the client are not served from the same domain? 

Other than that it looks good though.

**Comment by dspencer001** [comment]
Cole has said that the epub will always be on the same domain, so this should be fine.

**Pull request merged**